docs/installation/network-options.md

```diff
@@ -15,13 +15,21 @@ By default, K3s will run with flannel as the CNI, using VXLAN as the default bac
 The default backend for flannel is VXLAN. To enable encryption, pass the IPSec (Internet Protocol Security) or WireGuard options below.
 
 If you wish to use WireGuard as your flannel backend it may require additional kernel modules. Please see the [WireGuard Install Guide](https://www.wireguard.com/install/) for details. The WireGuard install steps will ensure the appropriate kernel modules are installed for your operating system. You need to install WireGuard on every node, both server and agents before attempting to leverage the WireGuard flannel backend option.
+The `wireguard` backend will be removed from v1.26 in favor of `wireguard-native` backend natively from flannel.
+
+We recommend that users migrate to the new backend as soon as possible. The migration requires a short period of downtime while nodes come up with the new configuration. You should follow these two steps:
+
+1 - Update the K3s config in all control-plane nodes. The config file `/etc/rancher/k3s/config.yaml` should include `flannel-backend: wireguard-native` instead of `flannel-backend: wireguard`
```

Comment:
```suggestion
1 - Update the K3s config in all control-plane nodes. The config file `/etc/rancher/k3s/config.yaml` should include `flannel-backend: wireguard-native` instead of `flannel-backend: wireguard`.
```